src/main/resources/config/liquibase/changelog/20211208041300_changelog.xml

```diff
@@ -0,0 +1,26 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <property name="autoIncrement" value="true"/>
+<changeSet id="20211208041300" author="julian-christl">
+    <!-- create new entity programming exercise test case -->
+    <createTable tableName="migration_changelog">
+        <column name="id" type="bigint" autoIncrement="${autoIncrement}">
```

Comment:
I don't really get why we sometimes use this notation, but I find the following more easy to understand

            <column autoIncrement="true" name="id" type="BIGINT">

Reply:
Because we don't have a standard and everyone just looks at old changelogs and some get a changelog with the boolean, some with the parameter.

I changed it.